jax2onnx/converter/ir_optimizations.py

```diff
@@ -439,6 +440,13 @@ def _replace_everywhere(
     old_name: Optional[str],
     new_v: "ir.Value",
 ) -> None:
+    if old_v is not None:
+        try:
+            ir_convenience.replace_all_uses_with(old_v, new_v)
+            return
+        except Exception:
+            # Fall back to manual path (legacy IR objects or pure-proto case)
+            pass
```

Comment:
If I understand it right you are working with protos anymore, so a fallback may not do anything I suppose